app/frontend/Shared/WebView.svelte

```diff
@@ -52,8 +52,34 @@
   function haveWebView () {
     webviewE.addEventListener("dom-ready", () => {
       dispatch("webview", webviewE);
+      webviewE.addEventListener("did-finish-load", resizeWebview());
     }, { once: true });
   }
+
+  const heightBuffer = 20;
+  async function resizeWebview () {
+    try {
+      const dimensions = await webviewE.executeJavaScript(`
+        document.body.style.minHeight = "0px";
+        document.body.style.height = "fit-content";
+        document.body.style.width = "fit-content";
+        new Promise(resolve => {
```

Comment:
```suggestion
        return new Promise(resolve => {
```

Does a `return` work here? Or does it break?
Was it implicit before?

Reply:
No, the `return` doesn't work. Your version was correct.